README.md

```diff
@@ -126,9 +126,9 @@ HLS should also just work.
 ### For WebAsm:
 
 ```sh
-nix-shell ~/ghc.nix --arg withWasiSDK true
+nix-shell ~/ghc.nix --arg withWasmSDK true
```

Comment:
`wasi-sdk` is the project name that provides the custom llvm build with wasm32-wasi sysroot, renaming it to `wasm-sdk` really adds confusion tbh

Reply:
Probably just rename it to `withWasm` and keep `withWasiSDK` for compat reasons for a while?